.github/pull_request_template.md

```diff
@@ -1,5 +1,35 @@
-# Proposed changes
+# Description
```

Comment:
Maybe we can remove this title and go strainght with the Fixes header? 

for title i would rather use something more descriptive

Reply:
I've reverted to `Proposed changes` as I think it's more inclusive of "fixes" + "desc" - I wanted to highlight the "description" part but we always add it, to it's not important to highlight it